hack/update-generated-completions.sh

```diff
@@ -4,9 +4,6 @@
 
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
-# ensure we have the latest compiled binaries
-"${OS_ROOT}/hack/build-go.sh"
```

Comment:
To be more clear -- `make update` depends on `make build`. Update does updates. Build does builds. This has been our convention for the entire lifetime of this project.